README.md

```diff
@@ -18,7 +18,7 @@ This will be the best place to discuss design and implementation.
 
 For sync communication we have a community slack with a #containerd channel that everyone is welcome to join and chat about development.
 
-**Slack:** https://community.docker.com/registrations/groups/4316
+**Slack:** http://dockr.ly/MeetUp
```

Comment:
if not, I'll show both URLs I think, the meetup in the short version could be confusing.

But yes with merging it,